commons/com.b2international.index/src/com/b2international/index/es/client/EsClusterStatus.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 B2i Healthcare Pte Ltd, http://b2i.sg
+ * Copyright 2023 B2i Healthcare Pte Ltd, http://b2i.sg
```

Comment:
The start date should be preserved in copyright notices.

Reply:
Three more files where the date was upgraded in place:

- CodeSystemUpgradeRestService
- AbstractSnomedExtensionApiTest
- ApiTestConstants

Reply:
License dates and since tags will be covered in a separate task before the release.